### PR TITLE
openvino_docs_MO_DG_prepare_model_convert_model_Converting_Model

### PR DESCRIPTION
**Pull request by msmykx-intel**
openvino_docs_MO_DG_prepare_model_convert_model_Converting_Model => mo-input-shapes

### CODE REVIEW

File: pages/documentation.rst [hunk 1]
```diff
@@ -24,8 +24,8 @@ Documentation
    :caption: Converting and Preparing Models
    :hidden:
 
-   openvino_docs_model_processing_introduction
-   openvino_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide
+   model-processing
+   mo-converting-models
```
Comment (sgolebiewski-intel) [inline pages/documentation.rst:28]: Do we all agree to use "mo-" prefix with the articles in this section?

Comment (kblaszczak-intel) [reply pages/documentation.rst:28]: I think we could skip it entirely if it didn't deserve being mentioned in the title, while in this case we can use the full name, like "model-optimizer-converting-models"

Comment (msmykx-intel) [reply pages/documentation.rst]: I changed prefixes from "mo-" to "-model-optimizer" in https://github.com/openvinotoolkit/openvino-docs/pull/18/commits/dcecfab80ebf40d80f894334f39e7c01f6f1acc3

Comment (sgolebiewski-intel) [reply pages/documentation.rst]: So I guess this issue is resolved ;)

**Review by sgolebiewski-intel — APPROVED** [review]
Looks good :)

**Pull request merged**